cms/static/js/spec/views/pages/group_configurations_spec.js

```diff
@@ -92,29 +92,22 @@ define([
                 this.view.render();
                 // We cannot use .toBeFocused due to flakiness.
                 expect($.fn.focus).toHaveBeenCalled();
-                expect(this.view.$(itemClassName)).toBeExpanded();
+                expect(this.view.$(groupConfigItemClassName)).toBeExpanded();
             });
 
             it('should not focus on any group configuration if url hash is empty', function() {
```

Comment:
I'm confused about what behavior these tests are verifying. @dan-f do you understand?